txpool/txpool.go

```diff
@@ -203,6 +204,31 @@ func (t *TxPool) addImpl(ctx string, tx *types.Transaction) error {
 	return nil
 }
 
+// GetTxs gets both queued and pending transactions
+func (t *TxPool) GetTxs() (map[types.Address]map[uint64]*types.Transaction, map[types.Address]map[uint64]*types.Transaction) {
+	queuedTxs := make(map[types.Address]map[uint64]*types.Transaction)
+	queue := t.queue
+	for addr, queuedTxn := range queue {
+		for _, tx := range queuedTxn.txs {
+			if _, ok := queuedTxs[addr]; !ok {
+				queuedTxs[addr] = make(map[uint64]*types.Transaction)
+			}
+			queuedTxs[addr][tx.Nonce] = tx
+		}
+	}
+
+	pendingTxs := make(map[types.Address]map[uint64]*types.Transaction)
+	sortedPricedTxs := t.sorted.index
+	for _, sortedPricedTx := range sortedPricedTxs {
+		if _, ok := pendingTxs[sortedPricedTx.from]; !ok {
+			pendingTxs[sortedPricedTx.from] = make(map[uint64]*types.Transaction)
+		}
+		pendingTxs[sortedPricedTx.from][sortedPricedTx.tx.Nonce] = sortedPricedTx.tx
+	}
+
+	return queuedTxs, pendingTxs
```

Comment:
You are returning `queuedTxs, pendingTxs` but in `txpool_endpoint.go` when you call this method you are assigning them in a different order: 
```
pendingTxs, queuedTxs := t.d.store.GetTxs()
```

Reply:
Thanks @Vuksan! Fixed this. The return is now pending followed by queued txs in all occurrences exactly how is done with geth (https://github.com/ethereum/go-ethereum/blob/d3e3a460ec947c9e1e963d1a35f887d95f23f99d/core/tx_pool.go#L482)